recipe/bld.bat

```diff
@@ -0,0 +1,9 @@
+mkdir build
+cd build
+
+REM Trick to avoid CMake/sh.exe error
+ren "C:\Program Files\Git\usr\bin\sh.exe" _sh.exe
+
+cmake -DCMAKE_INSTALL_PREFIX="%PREFIX%" -DCMAKE_INSTALL_LIBDIR="%LIBRARY_LIB%" -DCMAKE_INSTALL_INCLUDEDIR="%LIBRARY_INC%" -DCMAKE_INSTALL_BINDIR="%LIBRARY_BIN%" -DCMAKE_INSTALL_DATADIR="%LIBRARY_PREFIX%" ..
```

Comment:
Add `cmake -G "%CMAKE_GENERATOR%"`

Reply:
Also break this line into multiple lines with `^`

Reply:
Sure! I'm going to change that! thanks!